src/Raven.Client/Documents/Linq/RavenQueryProviderProcessor.cs

```diff
@@ -58,11 +58,15 @@ internal class RavenQueryProviderProcessor<T>
         private StringBuilder _declareBuilder;
         private DeclareToken _declareToken;
         private List<LoadToken> _loadTokens;
+        private HashSet<string> _loadAliases;
         private readonly HashSet<string> _loadAliasesMovedToOutputFuction;
         private int _insideLet = 0;
         private string _loadAlias;
         private bool _selectLoad;
         private const string DefaultLoadAlias = "__load";
+        private const string DefaultAlias = "__ravenDefaultAlias";
```

Comment:
I think it would be better to make it shorter e.g. `__alias` ?

Reply:
Also the variable would be better off by having `DefaultAliasPrefix` name instead of `DefaultAlias` as we append incremented number at the end